Sources/SwiftRoaring/RoaringBitmap.swift

```diff
@@ -72,41 +76,47 @@ public class RoaringBitmap: Sequence, Equatable, CustomStringConvertible,
     /// caller is
     /// responsible for memory management.
     ///
-    public func intersection(_ x: RoaringBitmap) -> RoaringBitmap {
-        return RoaringBitmap(ptr: croaring.roaring_bitmap_and(self.ptr, x.ptr))
+    @inlinable @inline(__always)
+    public func intersection(_ x: RoaringBitmap) -> Self {
+        return Self(ptr: croaring.roaring_bitmap_and(self.ptr, x.ptr))
```

Comment:
Here we should actually use `roaring_bitmap_and_inplace`. (Yes, I know that it was not done in the original code.)